src/esm_runscripts/filedicts.py

```diff
@@ -1258,3 +1267,66 @@ def resolve_file_movements(config: ConfigSetup) -> ConfigSetup:
     sim_file_collection = SimulationFileCollection.from_config(config)
     config = sim_file_collection.execute_filesystem_operation(config)
     return config
+
+
+def log_used_files(config: ConfigSetup) -> ConfigSetup:
+    """
+    Logs the files moved on this current phase.
+
+    Parameters
+    ----------
+    config : ConfigSetup
+        The complete simulation configuration.
+
+    Returns
+    -------
+    config : ConfigSetup
+        The complete simulation configuration, potentially modified.
+    """
+    if config["general"].get("verbose", False):
+        logger.info("\n::: Logging used files")
+
+    filetypes = config["general"]["relevant_filetypes"]
```

Comment:
To be discussed: If we use the Enum thing, we should use it everywhere

Reply:
`relevant_filtypes` is a list of files that is defined depending on the step `tidy`, `compute`, `preprocess`, ... (`grep -r relevant_filtypes src/esm_runscripts`). I suggest we don't touch it for consistency with previous versions. Also it seems natural to me that the relevant files for the given ESM-Tools phase are defined in the phase itself and not in `filedicts.py`.